api/openapi-spec/swagger.json

```diff
@@ -6846,6 +6846,10 @@
       "description": "Attach a volume as a floppy to the vmi.",
       "$ref": "#/definitions/v1.FloppyTarget"
      },
+     "io": {
+      "description": "IO specifies which QEMU disk IO mode should be used.",
+      "type": "string"
```

Comment:
@danielBelenky if this is just a string, we should at least add a validation webhook to check the values.

Reply:
Added validation. Thanks.